### ISSUE TITLE
Project development

### ISSUE THREAD

**Issue by cyberbudy** [maykinmedia/mobetta#23]
Is this project still alive? 
As I can see the last commit was almost a year ago and a lot of feature is still under development.

**Comment by sergei-maertens** [comment]
Yeah it is, but currently our resources go towards other projects that have higher prio
